packages/web-components/src/components/cbp-tooltip/cbp-tooltip.scss

```diff
@@ -0,0 +1,248 @@
+ * @Prop   --cbp-tooltip-control-focus-dark: 1px solid cyan;
+ * @Prop   --cbp-tooltip-padding: var(--cbp-space-inset-4x, 1rem);
+ * @Prop   --cbp-tooltip-gap: 0px;
+ * @Prop   --cbp-tooltip-offset: 0px;
+ * @Prop   --cbp-tooltip-link-decoration: none;
+ * @Prop   --cbp-tooltip-caret-size: 15px;
+ * @Prop   --cbp-tooltip-caret-offset: calc(var(--cbp-tooltip-caret-size) / 2);
+ * @Prop   --cbp-tooltip-color-hover: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-tooltip-color-hover-dark: var(--cbp-color-interactive-secondary-lighter);
+ * @Prop   --cbp-tooltip-color-bg-hover: var(--cbp-color-interactive-secondary-lighter);
+ * @Prop   --cbp-tooltip-color-bg-hover-dark: var(--cbp-color-interactive-disabled-light);
+ * @Prop   --cbp-tooltip-color-focus: var(--cbp-color-text-lightest);
+ * @Prop   --cbp-tooltip-color-focus-dark: var(--cbp-color-text-darkest);
+ * @Prop   --cbp-tooltip-color-bg-focus: var(--cbp-color-interactive-focus-dark);
+ * @Prop   --cbp-tooltip-color-bg-focus-dark: var(--cbp-color-interactive-focus-light);
+ */
+ :root{
+ 
+  --cbp-tooltip-color: var(--cbp-color-text-lightest);
+  --cbp-tooltip-color-dark: var(--cbp-color-text-darkest);
+  --cbp-tooltip-color-bg: var(--cbp-color-gray-cool-80);
+  --cbp-tooltip-color-bg-dark: var(--cbp-color-gray-cool-5);
+  --cbp-tooltip-gap: 0px;
+  --cbp-tooltip-offset: 0px;
+  --cbp-tooltip-link-decoration: none;
+  --cbp-tooltip-caret-size: 15px;
+  --cbp-tooltip-caret-offset: calc(var(--cbp-tooltip-caret-size) / 2);
+
+  --cbp-tooltip-color-hover: var(--cbp-color-interactive-secondary-darker);
+  --cbp-tooltip-color-hover-dark: var(--cbp-color-interactive-secondary-lighter);
+  --cbp-tooltip-color-bg-hover: var(--cbp-color-interactive-secondary-lighter);
+  --cbp-tooltip-color-bg-hover-dark: var(--cbp-color-interactive-disabled-light);
+
+  --cbp-tooltip-color-focus: var(--cbp-color-text-lightest);
+  --cbp-tooltip-color-focus-dark: var(--cbp-color-text-darkest);
+  --cbp-tooltip-color-bg-focus: var(--cbp-color-interactive-focus-dark);
+  --cbp-tooltip-color-bg-focus-dark: var(--cbp-color-interactive-focus-light);
+  
+}
+
+[data-cbp-theme=light] cbp-tooltip[context*=dark]:not([context=light-always]),
+[data-cbp-theme=dark] cbp-tooltip:not([context=dark-inverts]):not([context=light-always]) {
+
+  --cbp-tooltip-color-bg: var(--cbp-tooltip-color-bg-dark);
+  --cbp-tooltip-color: var(--cbp-tooltip-color-dark);
+ 
+  --cbp-tooltip-color-hover: var(--cbp-tooltip-color-hover-dark);
+  --cbp-tooltip-color-bg-hover: var(--cbp-tooltip-color-bg-hover-dark);
+
+  --cbp-tooltip-color-focus: var(--cbp-tooltip-color-focus-dark);
+  --cbp-tooltip-color-bg-focus: var(--cbp-tooltip-color-bg-focus-dark);
+}
+
+cbp-tooltip {
+  display: inline-block;
+  position: relative;
+  --cbp-tooltip-placement-gap: var(--cbp-tooltip-gap);
+  --cbp-tooltip-placement-offset: var(--cbp-tooltip-offset);
+  --cbp-link-decoration: var(--cbp-tooltip-link-decoration);
```

Comment:
this is not acting on a cbp-link (or should not be) component so it should not be named as such.

Reply:
removed this var, looks like it was set to none regardless, i think this was carry over from the original style sheet